apps/netsuite/tasks.py

```
@@ -1014,31 +1014,38 @@ def get_employee_expense_attribute(value: str, workspace_id: int) -> ExpenseAttr
             }
-        }
-        ExpenseAttribute.bulk_create_or_update_expense_attributes([attribute], 'EMPLOYEE', expense_group.workspace_id, True)
-        return get_employee_expense_attribute(expense_group.description.get('employee_email'), expense_group.workspace_id)
+            ExpenseAttribute.bulk_create_or_update_expense_attributes([attribute], 'EMPLOYEE', expense_group.workspace_id, True)
+            return get_employee_expense_attribute(expense_group.description.get('employee_email'), expense_group.workspace_id)
+    except (InvalidTokenError, InternalServerError) as e:
+        logger.info('Invalid Fyle refresh token or internal server error for workspace %s: %s', expense_group.workspace_id, str(e))
+        return None
 
-    return None
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)